ors-api/src/test/resources/application-test.yml

```diff
@@ -99,6 +99,9 @@ ors:
               radius: 150
               output_log: false
               log_location: ors/traffic_log
+          encoded_values:
```

Comment:
For the sake of backwards compatibility no `encoded_values` entries in the config should be necessary until the deprecation of extended storages has been completed and they are all removed  For now all the newly introduced EVs should be enabled based on the presence of `ext_storages.WayCategory`.

Please remove any new entries from the config file and instead make the necessary adaptations in the codebase.